tests/integ/ml/test_transformer.py

```diff
@@ -0,0 +1,524 @@
+#
+# Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
+#
+
+
+import pytest
+
+from snowflake.snowpark import Row
+from snowflake.snowpark.ml.transformer import (
+    Binarizer,
+    MinMaxScaler,
+    OneHotEncoder,
+    OrdinalEncoder,
+    StandardScaler,
+)
+from tests.utils import Utils
+
+
+def test_standardscaler(session):
+    df = session.create_dataframe(
+        [[1, 2, 3], [2, 3, 4], [4, 5, 6]], schema=["a", "b", "c"]
+    )
+    standarscaler = StandardScaler()
+    standarscaler.input_cols = ["A"]
+    standarscaler.output_cols = ["res"]
+    model = standarscaler.fit(df)
+
+    # test states table
+    df_table = session.table(model._states_table_name)
+    Utils.check_answer(
+        df_table,
+        [
+            Row(
+                STATES_A='{\n  "mean": 2.333333000000000e+00,\n  "stddev_pop": 1.247219307098796e+00\n}'
+            )
+        ],
+    )
+    res = model.transform(df)
+    Utils.check_answer(
+        res,
+        [
+            Row(A=1, B=2, C=3, RES=-1.069044547667817),
+            Row(A=2, B=3, C=4, RES=-0.26726093647105154),
+            Row(A=4, B=5, C=6, RES=1.3363062859224795),
+        ],
+    )
+
+
+def test_minmaxscaler(session):
+    df = session.create_dataframe(
+        [[1, 2, 3], [2, 3, 4], [4, 5, 6]], schema=["a", "b", "c"]
+    )
+    minmaxscaler = MinMaxScaler()
+    minmaxscaler.input_cols = ["A"]
+    minmaxscaler.output_cols = ["res"]
+    model = minmaxscaler.fit(df)
+
+    # test states table
+    df_table = session.table(model._states_table_name)
+    Utils.check_answer(df_table, [Row(STATES_A='{\n  "max": 4,\n  "min": 1\n}')])
+
+    res = model.transform(df)
+    Utils.check_answer(
+        res,
+        [
+            Row(A=1, B=2, C=3, RES=0.0),
+            Row(A=2, B=3, C=4, RES=0.3333333333333333),
+            Row(A=4, B=5, C=6, RES=1.0),
+        ],
+    )
+
+
+def test_merge_onehotencoder(session):
```

Comment:
```suggestion
def test_merge_onehot_encoder(session):
```
and the same for all other places in this file.